apps/shared/account/BUILD.bazel

```
@@ -0,0 +1,33 @@
+load("//tools:defaults.bzl", "ng_module")
+load("@io_bazel_rules_sass//:defs.bzl", "sass_binary")
+
+package(default_visibility = ["//visibility:private"])
+
+ng_module(
+    name = "account",
+    srcs = glob(
+        [
+            "*.ts",
+        ],
+        exclude = [
+            "*.spec.ts",
+        ],
+    ),
+    assets = [
+        "account.component.css",
```

Optional nit (if you agree): I think generated files should be specially-denoted (not feeling strongly)

```suggestion
        ":account.component.css",
```

---

I think thats a good idea to have that distinction.